app/src/main/res/values/styles.xml

```diff
@@ -7,4 +7,32 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="ButtonSet">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">75dp</item>
+        <item name="android:gravity">center</item>
+        <item name="android:background">#333333</item>
+        <item name="layout_constraintWidth_percent">0.24</item>
+        <item name="android:textColor">#fdfdfd</item>
+        <item name="android:textStyle">bold</item>
+        <item name="android:textSize">20sp</item>
+    </style>
+
+    <style name="ButtonDau">
+        <item name="android:layout_width">0dp</item>
+        <item name="android:layout_height">75dp</item>
+        <item name="android:gravity">center</item>
+        <item name="android:background">#ff9501</item>
```

Comment:
Màu sắc khai báo trong colors.xml 
Và đặt tên theo màu có color 
http://chir.ag/projects/name-that-color/#FF9501
``` suggestion
    <style name="ButtonDau">
        <item name="android:layout_width">0dp</item>
        <item name="android:layout_height">75dp</item>
        <item name="android:gravity">center</item>
        <item name="android:background">@color/color_pizazz</item>
```